.travis.yml

```diff
@@ -14,7 +14,8 @@ rvm:
   - 2.5.7
   - 2.6.3
   - 2.6.5
-  - 2.7.0
+  - 2.7.2
```

Comment:
It would be nice to include `ruby-head` into the testing. That way, similar issues could be cauht earlier.

Reply:
Sadly travis is alway behind with Ruby releases. It does not yet support Ruby 3.0.0. Tried it and the Ruby 3.0.0 could not be found. It does make sense to use head though.

Reply:
Update on that. I see ruby-head is pointing to Ruby 3.1.0 but the image does not include make if I am reading the error message correctly. Will set aside that option for a separate branch. If you feel so inclined to update the .travis.yml file to get it working I'd be glad to review a PR for that.